data/campaigns/campaigns.ts

```diff
@@ -17,59 +17,81 @@ export const ALWAYS_SHOW = 0;
  */
 const CAMPAIGNS = {
   'nwac-campaign-q4-2023': {
-    startDate: new Date('2023-12-01'),
-    endDate: new Date('2024-01-01'),
     enabled: true,
     allowedCenters: ['NWAC'],
     locations: {
       'observation-list-view': {
         frequency: ALWAYS_SHOW,
+        startDate: new Date('2023-12-01'),
+        endDate: new Date('2024-01-01'),
       },
       'map-view': {
-        frequency: 8 * 60 * 60 * 1000, // 8 hours
+        frequency: 16 * 60 * 60 * 1000, // 16 hours
+        startDate: new Date('2023-12-22'),
+        endDate: new Date('2024-01-01'),
+      },
+    },
+  },
+
+  'sac-campaign-q4-2023': {
+    enabled: true,
+    allowedCenters: ['SNFAC'],
```

Comment:
TBH this seems like a mismatch between intent and structure - do we have a use-case for a campaign to be shown for more than one center?